Microsoft.Toolkit.Uwp.UI/Cache/CacheBase.cs

```diff
@@ -233,6 +233,29 @@ public Task<T> GetFromCacheAsync(Uri uri, bool throwOnError = false, Cancellatio
         }
 
         /// <summary>
+        /// Retrieves item represented by Uri from the in-memory cache if it exists and is not out of date.
```

Comment:
I would add that if one doesn't exist, we will return the default instance of this type.